Rubberduck.UnitTesting/CodeGeneration/ITestCodeGenerator.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Rubberduck.Parsing.Symbols;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+
+namespace Rubberduck.UnitTesting.CodeGeneration
+{
+    public interface ITestCodeGenerator
+    {
+        void AddTestModuleToProject(IVBProject project);
+        void AddTestModuleToProject(IVBProject project, Declaration stubSource);
+        string GetNewTestMethodCode(IVBComponent component);
+        string GetNewTestMethodCodeErrorExpected(IVBComponent component);
```

Comment:
FWIW - those 2 methods could be consolidated with a 2nd parameter -- perhaps an enum -- to make it easier to extend the stubs beyond the default & expected error templates.

Reply:
I could see that, although a two member enum would strike me as a bit weird.  IIR this was the existing naming (or close), but I think if this were extended an enum might be in order.